docs/graphql-api.md

```diff
@@ -0,0 +1,184 @@
+---
+title: GraphQL API
+keywords: homepage
+sidebar: talk_sidebar
+permalink: /graphql-api.html
+summary:
+---
+
+## GraphiQL
+
+Explore the GraphQL API at
+
+http://localhost:3000/api/v1/graph/iql
+
+with the “Documentation Explorer” on the right side.
+
+
+![GraphiQL screenshot](https://user-images.githubusercontent.com/815894/28073533-47debff6-6656-11e7-82fb-eb5aebefa350.png)
+
+
+
+
+
+After logged-in as admin at admin/configure you are able to use GrapiQL to show admin/user data.
+
+
+
+## First sample query
```

Comment:
Instead of providing examples here on sample queries, I would direct users to http://graphql.org/learn/ to learn how to make requests using GraphQL because as mentioned before, having concrete examples here make it difficult to keep up to date.

Suggestions: remove sample query/responses, direct to http://graphql.org/learn/ instead.